test_requirements.txt

```diff
@@ -5,6 +5,7 @@ pytest-pep8
 pytest-pylint
 pytest-cov
 pytest-django
+pytest-capturelog
```

Comment:
Just by being installed it gets used, so that when tests fail anything that went to the python logger goes to stdout.  This is something baked into nose, that pytest surprisingly didn't have out of the box.